src/bdfsclient/ActionHandler.cpp

```diff
@@ -47,7 +48,11 @@ namespace dfs
     {"/dev/nbd0",false},
     {"/dev/nbd1",false},
     {"/dev/nbd2",false},
-    {"/dev/nbd3",false}
+    {"/dev/nbd3",false},
```

Comment:
Either we should read this from some nbd api interface if any, or we should list /dev/nbd*